use_cases/eluc/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN pip install --no-cache-dir --upgrade pip && \
 # Copy source files over
 COPY . .
 
+# Python setup script - downloads data and processes it
+RUN python -m app.process_data
```

Comment:
Change our dockerfile to download the data from HuggingFace before we start the app so that we don't have to push the csv to the git repo

Reply:
How fast (or slow) is it?

Reply:
It takes about a minute.